route/osrm/client.go

```diff
@@ -128,7 +128,7 @@ func (c *client) get(uri string) (data []byte, err error) {
 	var key string
 
 	if c.useCache {
-		key = fmt.Sprintf("%x", sha1.Sum([]byte(uri)))
+		key = fmt.Sprintf("%x", sha256.Sum256([]byte(uri)))
```

Comment:
Same here. Speed matters the most in this case and this is not a security relevant operation. We are creating a key for a cache. With that said `sha1` is probably also not the best (fastest) choice, but I do not see the need to use `sha256`.

Reply:
Revert to sha1 and add #nosec after discussion.